signature-aggregator/aggregator/aggregator.go

```diff
@@ -64,7 +64,7 @@ var (
 )
 
 type SignatureAggregator struct {
-	network                peers.AppRequestNetwork
+	network                peers.AppRequestNetworkTestHelper
```

Comment:
nit: I know that this is temporary but I don't think that we should be merging anything named `TestHelper` into main in the critical code path. If we are temporarily keeping the interface I would still call it something else.